modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc

```diff
@@ -40,13 +40,16 @@ using apollo::common::math::LineSegment2d;
 using apollo::common::math::NormalizeAngle;
 using apollo::common::math::Vec2d;
 
-IterativeAnchoringSmoother::IterativeAnchoringSmoother() {
+IterativeAnchoringSmoother::IterativeAnchoringSmoother(
+    const PlannerOpenSpaceConfig& planner_open_space_config) {
+  // TODO(Jinyun, Yu): refactor after stablized.
   const auto& vehicle_param =
       common::VehicleConfigHelper::Instance()->GetConfig().vehicle_param();
   ego_length_ = vehicle_param.length();
   ego_width_ = vehicle_param.width();
   center_shift_distance_ =
       ego_length_ / 2.0 - vehicle_param.back_edge_to_center();
+  planner_open_space_config_.CopyFrom(planner_open_space_config);
```

Comment:
`planner_open_space_config_ = planner_open_space_config;` could be more efficient.